internal/api/grpc/server/middleware/execution_interceptor.go

```diff
@@ -80,78 +83,72 @@ func executeTargetsForResponse(ctx context.Context, targets []execution.Target,
 var _ execution.ContextInfo = &ContextInfoRequest{}
 
 type ContextInfoRequest struct {
-	FullMethod string      `json:"fullMethod,omitempty"`
-	InstanceID string      `json:"instanceID,omitempty"`
-	OrgID      string      `json:"orgID,omitempty"`
-	ProjectID  string      `json:"projectID,omitempty"`
-	UserID     string      `json:"userID,omitempty"`
-	Request    interface{} `json:"request,omitempty"`
+	FullMethod      string  `json:"fullMethod,omitempty"`
+	InstanceID      string  `json:"instanceID,omitempty"`
+	OrgID           string  `json:"orgID,omitempty"`
+	ProjectID       string  `json:"projectID,omitempty"`
+	UserID          string  `json:"userID,omitempty"`
+	Request         Message `json:"request,omitempty"`
+	originalRequest interface{}
```

Comment:
I like the wrapper type for the marshalling, but it has to be used also in the unmarshalling properly otherwise you'll run into the same issues again and you then do not even need the originalRequest
```suggestion
	Request         Message `json:"request,omitempty"`
```

Reply:
Changed